waspc/data/Generator/templates/server/src/auth/providers/config/github.ts

```diff
@@ -80,10 +80,8 @@ const _waspConfig: ProviderConfig = {
             stateTypes: ['state'],
             userSignupFields: _waspUserSignupFields,
             getAuthorizationUrl: ({ state }) => github.createAuthorizationURL(state, config),
-            getProviderInfo: async ({ code }) => {
-                const { accessToken } = await github.validateAuthorizationCode(code);
-                return getGithubProfile(accessToken);
-            },
+            getProviderTokens: ({ code }) => github.validateAuthorizationCode(code),
```

Comment:
Refactor OAuth a bit to split getting tokens and getting user profile into two steps so it's easier to write a hook `onAfterOAuthTokenReceived`